dev/jobs/prep.sh

```diff
@@ -96,30 +103,38 @@ if [[ ${PROCESS_TROPCY} = "YES" ]]; then
     fi
 
 else
-    if [[ ${ROTDIR_DUMP} = "NO" ]]; then cp "${COMIN_OBSDMP}/${RUN_local}.t${cyc}z.syndata.tcvitals.tm00" "${COMOUT_OBS}/"; fi
+    if [[ ${ROTDIR_DUMP} = "NO" ]]; then
+       cpfs "${COMINobsproc}/${RUN_local}.t${cyc}z.syndata.tcvitals.tm00" "${COMOUT_OBS}/"
+    fi
 fi
 
 
 ###############################################################
 # Generate prepbufr files from dumps or copy from OPS
-if [[ ${MAKE_PREPBUFR} = "YES" ]]; then
-    if [[ ${ROTDIR_DUMP} = "YES" ]]; then
+if [[ ${MAKE_PREPBUFR:-"YES"} = "YES" ]]; then
```

Comment:
Default should be `NO`, IMO.  Unless asked to create, let it copy the one from dump archive.

Reply:
No we want a prepbufr generated within the experiment because the prepbufr generation uses guess from the prior gdas cycle as input. If you pull in the prepbufr from the GDA (ops v16) then it will essentially wipe out the prior guess and introduce a prepbufr generated with guess from ops. We always want a prepbufr generated if we're cycling.
```
kate.friedman@clogin02:/lfs/h2/emc/ptmp/kate.friedman/comrot/RUNTESTS/COMROOT/C96_atm3DVar_extended> grep gdas logs/2021122100/gfs_prep.log | grep cp
+ 2.119s + cp /lfs/h2/emc/ptmp/kate.friedman/comrot/RUNTESTS/COMROOT/C96_atm3DVar_extended/gdas.20211220/18//model/atmos/history/gdas.t18z.atmf006.nc sgesprep
```

Reply:
`grep`ping through obsproc and prepobs, I don't see references to `MAKE_PREPBUFR`.  I do see `MAKE_NSSTBUFR`, though.  I think `MAKE_PREPBUFR` is now assumed `YES`.  The `JOBSPROC_GLOBAL_PREP` job calls `exglobal_makeprepbufr.sh` without an `if` block.  We could consider removing it from the workflow.

Reply:
Ah, thanks for checking @DavidHuber-NOAA ! I'll remove it, thanks!

Reply:
Removed `MAKE_PREPBUFR` switch and use @ 538218478.